src/Orleans/Configuration/GlobalConfiguration.cs

```diff
@@ -449,6 +451,10 @@ internal void SetReminderServiceType(ReminderServiceProviderType reminderType)
 
         public string DefaultPlacementStrategy { get; set; }
 
+        public VersionCompatibilityStrategy DefaultVersionCompatibilityStrategy { get; set; }
```

Comment:
I suggest we remove "Version" from the names to make them shorter or at least replace it with "Grain".